api/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use LAMP image
+FROM mattrayner/lamp:latest-1804
+
+# Install package dependencies
+RUN apt-get update
+RUN apt-get -y install libapache2-mod-php7.4
+RUN apt-get -y install php7.4-mysql
+RUN apt-get -y install php7.4-ldap
+RUN apt-get -y install php7.4-intl
+
+# Install vendor dependencies
+ADD . /api
+WORKDIR /api
```

Comment:
I'd suggest moving the WORKDIR command above the ADD command.

Reply:
done